Source/Task/Callback_STL.h

```diff
@@ -41,19 +43,14 @@ class Callback
     {
         Clear();
     }
-    
-    // Disable copy ctor and assignment, as these cannot be implemented without 
-    // potentially throwing exceptions
-    Callback<CallbackType, CallbackDataType, CallbackThunk>(const Callback<CallbackType, CallbackDataType, CallbackThunk>&) = delete;
```

Comment:
Default copy/assignment doesn't work in this class because there is no copy of the LIST_ENTRY memory.  You're leaving yourself open to crashes.  If you just removed because delete isn't supported on Android consider just adding these as private decls with no implementation.

Reply:
Hmm, I'm not sure why Luca removed this, it seems to compile fine for me on Android with it. I'll add it back.